smart-contracts/test/Unlock/erc20.js

```diff
@@ -0,0 +1,22 @@
+const unlockContract = artifacts.require('Unlock.sol')
```

Comment:
I've realized that this line could actually be in `proxy.js`. Not for this PR though.